roles/edpm_telemetry/molecule/default/prepare.yml

```diff
@@ -200,3 +200,16 @@
               }
             ]
           }
+    - name: Create 10-telemetry.conf
+      ansible.builtin.copy:
+        dest: /etc/rsyslog.d/10-telemetry.conf
+        mode: '0644'
+        owner: root
+        group: root
+        content: |
+          module(load="imjournal" PersistStateInterval="10"
+                StateFile="journal_state")
+
+          action(type="omfwd" target="172.17.0.80" port="10514" protocol="tcp"
```

Comment:
I don't know if i like these parameters hardcoded here. What about define them as vars with these values as defaults?

Reply:
This is preparing everything for the test. The file will be configured because it will be copied directly from the control plane to the compute node. What I am trying to do here is to have a valid file that wont fail during the test. The values here are irrelevant. I don't really see the need to move them to variables, they would only be testing variables for some feature (writing into a remote server) that wont be tested as it is out of ansible scope.

Reply:
sorry @jlarriba , my bad... i missed the fact that it was in molecule testing :)